data/organizations.json

```diff
@@ -22,6 +22,12 @@
     "Meaning": "Collection of free software, a license",
     "URL": "https://gnu.org",
     "What they do": "program of FSF, develops many linux tools and a commonly used license"
+  },
+   {
+    "Abbreviation": "BCG",
+    "Meaning": "Boston Consulting Group",
+    "URL": "https://www.bcg.com/",
+    "BCG is a global consulting firm that partners with leaders in business and society to tackle their most important challenges"
```

Comment:
Hello,

It seems that the property's key is missing. Do you have an example of an usage of the acronym BCG in the FOSS community?